backend/django/core/templates/projects/create/create_wizard_labels.html

```diff
@@ -31,7 +37,13 @@ <h3>Instructions</h3>
             <li class="list-group-item">If you plan on uploading a data file that contains labels, the label categories in the file must match those provided on this page.</li>
             <li class="list-group-item">You cannot update the labels for a project after the project is created.</li>
           </ul>
-          <table class="table table-striped">
+          <div class="form-group">
+            <label class="control-label" for="{{ wizard.form.data.id_for_label }}">You may also upload a .csv file containing labels and their descriptions (label and description columns required):</label>
```

Comment:
Note: when this happens any existing labels in the label fields are overwritten by the file. This isn't wrong, but we may want to note that here.

Reply:
added